pkg/rancher-desktop/pages/FirstRun.vue

```diff
@@ -92,7 +92,7 @@ import _ from 'lodash';
 import Vue from 'vue';
 import { mapGetters } from 'vuex';
 
-import { firstStableVersion } from '@pkg/backend/k3sHelper';
+import { highestStableVersion } from '@pkg/backend/k3sHelper';
```

Comment:
I believe this import to `k3sHelper` (which first originated from #7180) is causing this error when running from a first run:

![image](https://github.com/user-attachments/assets/25a46ecc-0b3b-452b-8e75-ae4d1fda4b45)

Reply:
Yes, will be fixed by #7249